configuration.nix

```diff
@@ -55,6 +55,7 @@ isMaximal: {
       sql.enable = isMaximal;
       java.enable = isMaximal;
       kotlin.enable = isMaximal;
+      haskell.enable = isMaximal;
```

Comment:
Haskell tooling is annoyingly large, and Haskell is not a language common enough to justify pulling said tooling on maximal configuration.

Reply:
Disabled now.